udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +90,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
+   * Asks for 1 comment if receives non-positive input.
+   * Returns null on error.
    */
+  private Integer getCommentCount(HttpServletRequest request) {
+    // Get the input from the form.
+    String commentCountString = request.getParameter("comment-count");
+
+    // Convert the input to an int.
+    Integer commentCount;
+    try {
+      commentCount = Integer.parseInt(commentCountString);
 
-  private String getRequestParameter(HttpServletRequest request, String comment, String defaultValue) {
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + commentCountString);
+      return null;
+    }
+    if (commentCount <= 0)
+      return 1;
+    else
+      return commentCount;
```

Comment:
Style requires braces, even when optional:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
Will be changed in APIs PR

Reply:
Changed in APIs PR